### DISCUSSION TITLE
Removing snapshot version from Micronaut Launch

### DISCUSSION THREAD

**Discussion by sdelamo** [micronaut-projects/micronaut-starter#2109] [Q&A]
I think we should remove the snapshot version from Micronaut Launch. It adds more confusion than benefit. It sometimes gets updated when we release a new version, and the same stable version number is shown twice.  

![CleanShot 2023-10-04 at 07 35 08@2x](https://github.com/micronaut-projects/micronaut-starter/assets/864788/05c15eec-94c5-40b9-bae4-901028602efa)

Thoughts?

**Comment by sdelamo** [comment]
ping @micronaut-projects/core-developers

**Comment by timyates** [comment]
I agree we should drop the snapshot version

Not sure what should replace it...or if we should just show the latest 4.x and the latest 3.x

**Comment by wetted** [comment]
I can't say I've ever needed the snapshot version from Launch. I agree with dropping it.
